matrix_client/client.py

```diff
@@ -127,6 +127,10 @@ def start_listener_thread(self, timeout=30000):
             e = sys.exc_info()[0]
             print("Error: unable to start thread. " + str(e))
 
+    def upload(self,content,content_type):
+        response = self.api.media_upload(content,content_type)["content_uri"]
```

Comment:
If `content_uri` doesn't exist for whatever reason this will not fail gracefully (with a nasty `KeyError` rather than with `MatrixRequestError` or some other well-defined exception). We should really be aiming to meet the terms of our API contract regardless of what stuff is thrown at us by the homeserver.